src/components/DogForm.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import styled from 'styled-components';
+
+function DogForm({
+  dog,
+  dog: { name, bio, age, breed, image },
+  setDog,
+  createNewDog,
+  edit,
+  handleUpdate,
+}) {
+  const updateDog = (att, val) => {
+    const newDog = { ...dog, [att]: val };
+    setDog(newDog);
+  };
```

Comment:
nice work with this function to accept multiple props and reducing the lines of code you're writing